pandas/tests/groupby/test_categorical.py

```diff
@@ -48,8 +48,9 @@ def setUp(self):
              'D': np.array(
                  np.random.randn(8), dtype='float32')})
 
-        index = MultiIndex(levels=[['foo', 'bar', 'baz', 'qux'], ['one', 'two',
-                                                                  'three']],
+        index = MultiIndex(levels=[pd.CategoricalIndex(
```

Comment:
Well my test is the only one using `self.mframe` - in fact quite a few tests could use `self.mframe` now. I can change it back, but this feels more natural.